### PR TITLE
refactor(ui): remove custom button component and replaced it with antd button

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:
I worked on refactoring, removed custom button component and replaced it with antd buttons

image
image
image
image
image
image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by Sachin-chaurasiya — DISMISSED** [review]
Looks Good 👍

File: openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx [hunk 1]
```diff
@@ -11,63 +11,67 @@
...
+    <div data-testid="inline-edit">
+      {children}
+      <button data-testid="cancel" onClick={onCancel}>
+        cancel
+      </button>
+      <button data-testid="save" onClick={onSave}>
+        save
+      </button>
+    </div>
+  ));
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx:41]: We are rewriting the whole component here and testing it again which is not required. 
We can simply mock it and check whether it's being rendered or not that's it

File: openmetadata-ui/src/main/resources/ui/src/components/common/success-screen/SuccessScreen.tsx [hunk 2]
```diff
@@ -126,14 +125,11 @@ const SuccessScreen = ({
             {!isUndefined(fetchAirflowStatus) && (
               <div className="tw-flex-none">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/success-screen/SuccessScreen.tsx:126]:
```suggestion
              <div className="flex-none">
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/success-screen/SuccessScreen.tsx [hunk 3]
```diff
@@ -126,14 +125,11 @@ const SuccessScreen = ({
...
                 <Button
-                  className={classNames('tw-self-center tw-py-1 tw-px-1.5', {
-                    'tw-opacity-40': isFetchingStatus,
-                  })}
+                  ghost
                   data-testid="airflow-status-check"
                   disabled={isFetchingStatus}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/success-screen/SuccessScreen.tsx:130]:
```suggestion
                  loading={isFetchingStatus}
```

https://ant.design/~demos/components-button-demo-loading

File: openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx:41]: For cancel we can avoid but with the save button, we are testing onSave function that's why we need the save button in mock component

**Review by chirag-madlani — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx]: For future references:
Unit tests only ment to tests components on isolation.
No end to end tests or user flow testing can covered in unit tests 😅

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyInput.test.tsx]: sure, thanks you 👍

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**